python/ray/data/tests/test_raydp_dataset.py

```diff
@@ -46,6 +46,13 @@ def test_raydp_to_spark(spark_on_ray_small):
     rows = [r.value for r in df.take(5)]
     assert values == rows
 
+def test_spark_on_ray(spark_on_ray_small):
```

Comment:
nit: can you name the test consistently with the tests above (test_raydp_xxxx)?

Reply:
Thanks, I've fixed it.